twilight-mention/src/parse/impl.rs

```diff
@@ -84,6 +86,132 @@ impl ParseMention for Id<ChannelMarker> {
     }
 }
 
+impl ParseMention for CommandMention {
+    const SIGILS: &'static [&'static str] = &["/"];
```

Comment:
i dont think we have to provide the lifetime explicitly here

Reply:
Other implementations do this too.